artemis-core-client/src/main/java/org/apache/activemq/artemis/core/message/impl/CoreMessage.java

```diff
@@ -1070,26 +1102,22 @@ public CoreMessage putObjectProperty(final String key, final Object value) throw
 
    @Override
    public Short getShortProperty(final SimpleString key) throws ActiveMQPropertyConversionException {
-      checkProperties();
-      return properties.getShortProperty(key);
+      return checkProperties().getShortProperty(key);
    }
 
    @Override
    public Short getShortProperty(final String key) throws ActiveMQPropertyConversionException {
-      checkProperties();
-      return properties.getShortProperty(SimpleString.toSimpleString(key, getPropertyKeysPool()));
+      return checkProperties().getShortProperty(SimpleString.toSimpleString(key, getPropertyKeysPool()));
```

Comment:
delegate to 
   public Short getShortProperty(final SimpleString key) throws ActiveMQPropertyConversionException {